kratos/utilities/merge_variable_lists_utility.h

```diff
@@ -0,0 +1,269 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+	           
+#if !defined(KRATOS_MERGE_VARIABLE_LISTS_H_INCLUDED )
+#define  KRATOS_MERGE_VARIABLE_LISTS_H_INCLUDED
+
+
+
+// System includes
+#include <string>
+#include <iostream> 
+
+
+// External includes 
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+  ///@addtogroup ApplicationNameApplication
+  ///@{
+
+  ///@name Kratos Globals
+  ///@{ 
+  
+  ///@} 
+  ///@name Type Definitions
+  ///@{ 
+  
+  ///@} 
+  ///@name  Enum's
+  ///@{
+      
+  ///@}
+  ///@name  Functions 
+  ///@{
+      
+  ///@}
+  ///@name Kratos Classes
+  ///@{
+  
+  /// Short class definition.
+  /** Detail class definition.
+  */
+  class MergeVariableListsUtility
```

Comment:
- please add some documentation
- and maybe a test
- splitting into header&source?
- there are leftover debugging statements

Reply:
Sure! (This was done in a rush by @RiccardoRossi to make things work)

Reply:
documentation was added as well as a test. the leftover debugging output was removed.

with regards to splitting in header and source i don't really see the need of it...it is a rather small function... do you see it really important?